concent_api/core/tests/test_auth_force_payment.py

```diff
@@ -131,3 +136,6 @@ def test_provider_send_force_payment_and_concent_should_return_it_to_requestor_w
         self._assert_stored_message_counter_not_increased()
 
         self._assert_client_count_is_equal(1)
+
+        last_pending_message = PendingResponse.objects.filter(delivered = False).last()
+        assert last_pending_message is None
```

Comment:
In tests, when you check results, I would use `self.assert*` from `unittest` or `assert_that` form `assertpy`

Reply:
Done.